### DISCUSSION TITLE
Is wayland support planned?

### DISCUSSION THREAD

**Discussion by envyniv** [thevindu-w/clip_share_server#20] [Ideas]
I love this nifty utility. I was wondering, is wayland support planned?

**Comment by thevindu-w** [comment]
Everything is already working* on Wayland with Xwayland except get screenshot. But there's a workaround for that too. You can manually take the screenshot and copy it to clipboard (probably you can do it with a single keyboard shortcut), and then get the copied image in the same way you would get the screenshot. Since the copied image is the screenshot, you get the screenshot indirectly.
The reason behind screenshots not working directly is the Wayland security feature. That breaks many screenshot applications on Linux :(

*Though I didn't check it recently on Wayland, it worked some time back on Wayland and I think it should be still working.

If you find any other functionality not working, please specify that. It would be helpful for me to identify that and fix it.
